webui/i18n/de.json

```diff
@@ -0,0 +1,51 @@
+{
+  "Language": "German",
+  "Translation": {
+    "Video Script Settings": "**Video Einstellungen**",
+    "Video Subject": "Worum soll es in dem Video gehen? (Geben Sie ein Keyword an, :red[Dank KI wird automatisch ein Drehbuch generieren])",
+    "Script Language": "Sprache zum Generieren von Drehbüchern (KI gibt automatisch basierend auf der Sprache Ihres Keywords aus)",
+    "Generate Video Script and Keywords": "Klicken Sie hier, um mithilfe von KI ein „Drehbuch“ und „Video-Keywords“ basierend auf dem **Keyword** zu generieren.",
```

Comment:
I am not sure if this characters need to be changed. I see that they are kept in the other translations as well as in one line of your translation. Maybe to keep it consistent it could be better to keep them like they are in the other languages. What do you think?

![image](https://github.com/harry0703/MoneyPrinterTurbo/assets/107332190/ff549f1e-4563-410a-9147-96fb931bfd5c)

Reply:
@danicruz0415 I see, you're right I've fixed that :)
For consistency I should also use the parentheses

Thanks!